fasst_service_sample/package.json

```diff
@@ -91,8 +93,11 @@
     "eslint": "^7.15.0",
     "find": "^0.3.0",
     "fs-extra": "^9.0.1",
-    "jasmine": "^3.6.3",
+    "mocha": "^8.2.1",
+    "chai": "^4.2.0",
+    "@types/chai": "^4.2.14",
     "nodemon": "^2.0.6",
+    "should": "^13.2.3",
```

Comment:
à quoi corresponds la lib should?

Reply:
j'ai trouvé quand j'ai commencé à regarder les tutos sur les tests, qqchose pour les assertion, mais oui, comme je ne l'utilise pas, je vait la uninstall